filters/openpolicyagent/openpolicyagent.go

```diff
@@ -79,6 +79,7 @@ type OpenPolicyAgentRegistry struct {
 	reuseDuration          time.Duration
 	cleanInterval          time.Duration
 	instanceStartupTimeout time.Duration
+	config                 *OpenPolicyAgentInstanceConfig
```

Comment:
nitpick: do we want to make it transparent that this does not hold the complete configuration for an instance but the general config that is the same for all instances?